quay-enterprise/custom-cert-kube-apps.md

```diff
@@ -1,10 +1,8 @@
 ## Using Helm Registry Plug-in with a self-signed certificate
 
+This document assumes you have deployed [Quay Enterprise with a self-signed certificate.][self-signed]
 
-
-This document assumes you have deployed Quay Enterprise with a self-signed certificate.
-
-[Appr](https://github.com/app-registry/appr) makes use of the python requests library, which by default only trusts a standard set of certificates. This prevents helm from interacting with a Quay Enterprise instance that is using a self-signed certificate.
+[Appr](https://github.com/app-registry/appr) uses a [Python Requests library][python-requests], which trusts only a standard set of certificates by default. This prevents Helm from interacting with a Quay Enterprise instance that is using a self-signed certificate.
```

Comment:
missed this - sorry. change to [Appr][appr], and add the link at the bottom of the page.